cmd/skywire-cli/commands/vpn/vvpn.go

```diff
@@ -2,33 +2,47 @@ package clivpn
 
 import (
 	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
 
 	"fmt"
 	"log"
 	"os"
-	"strings"
+
 	"github.com/spf13/cobra"
 	"github.com/toqueteos/webbrowser"
-
+	utilenv "github.com/skycoin/skywire-utilities/pkg/skyenv"
+	skyenv "github.com/skycoin/skywire/pkg/skyenv"
 	"github.com/skycoin/skywire/pkg/visor/visorconfig"
 	clirpc "github.com/skycoin/skywire/cmd/skywire-cli/commands/rpc"
```

Comment:
Also, is the file name intentional?

Reply:
it was an attempt to avoid confusion with any other package named rpc and specify that this is the one that is associated with or provided by the cli subcommands

that same rpc client function was in a few different files